go.mod

```diff
@@ -26,6 +25,7 @@ require (
 	k8s.io/apimachinery v0.19.0
 	k8s.io/client-go v0.19.0
 	k8s.io/code-generator v0.19.0
+	k8s.io/klog v1.0.0
```

Comment:
We want to only use `klog/v2` now, just need to replace all imports `k8s.io/klog` with `k8s.io/klog/v2`, can you ensure this is done?